### PR TITLE
Create Wallets Endpoint

### PR DESCRIPTION
**Pull request by ConnorLengyel27**

### CODE REVIEW

**Comment by luis-grados** [comment]
@ConnorLengyel27 , sorry for adding this now, can we have a get all for the wallets? Something like /wallets in the controller. Similar to what I have in this PR: https://github.com/ucmo-cs/Carebears_SP23/pull/20/files -> wallet-add.component.ts line 16. However, I guess we'll need a parameter for petID so that it pulls all wallets but for an specific pet. Check Discord #general.

File: docker/wallets.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+USE PETVAX;
+
+CREATE TABLE wallets (
+  walletId char(99) NOT NULL,
+  petId char(99) NOT NULL,
+  active tinyint(1),
+  PRIMARY KEY (`walletId`)
+);
+
+INSERT INTO `wallets` (`walletId`, `petId`, active) VALUES ('969997a6-d414-11ed-afa1-0242ac120002', 'a90a655e-b25b-11ed-8531-0242ac120002', 1);
```
Comment (cmn56690) [inline docker/wallets.sql]: active needs tick marks around it

**Comment by cmn56690** [comment]
> @ConnorLengyel27 , sorry for adding this now, can we have a get all for the wallets? Something like /wallets in the controller. Similar to what I have in this PR: https://github.com/ucmo-cs/Carebears_SP23/pull/20/files -> wallet-add.component.ts line 16. However, I guess we'll need a parameter for petID so that it pulls all wallets but for an specific pet. Check Discord #general.

@luis-grados, you're asking for /wallets?active=true with a cookie of petId, correct?

**Comment by ConnorLengyel27** [comment]
Updated pull request to allow for getting wallets by petId

**Review by cmn56690 — APPROVED** [review]

**Pull request merged**